clients/admin-ui/src/features/datastore-connections/ConnectionsHeader.tsx

```diff
@@ -1,21 +1,27 @@
-import { Flex, Heading, Spacer } from "@fidesui/react";
+import { Box, Flex, Heading, Link, Text } from "@fidesui/react";
 import React from "react";
 
-import AddConnectionButton from "./add-connection/AddConnectionButton";
+import { ADD_SYSTEMS_ROUTE } from "../common/nav/v2/routes";
 
-type ConnectionsHeaderProps = {
-  hasConnections: boolean;
-};
-
-const ConnectionsHeader: React.FC<ConnectionsHeaderProps> = ({
-  hasConnections = false,
-}) => (
-  <Flex mb="24px" justifyContent="center" alignItems="center">
-    <Heading fontSize="2xl" fontWeight="semibold">
-      Connections
+const ConnectionsHeader: React.FC = () => (
+  <Flex
+    mb="24px"
+    flexDirection="column"
+    justifyContent="center"
+    alignItems="left"
+  >
+    <Heading marginBottom={4} fontSize="2xl" fontWeight="semibold">
+      Unlinked Connections
     </Heading>
-    <Spacer />
-    {hasConnections && <AddConnectionButton />}
+    <Box maxWidth="450px">
+      <Text fontSize="sm">
+        Connections are now created in the{" "}
+        <Link color="complimentary.500" href={ADD_SYSTEMS_ROUTE}>
+          system configuration
+        </Link>{" "}
+        section. You can link existing connections to a new or existing system.
```

Comment:
Can you add a feature flag that gives us the ability to toggle the add connection button on and off? I think this is going to be required or else Roger won't be able to QA the ability to link unlinked connectors to a system. He won't be able to create any to add 😅


You can add a new flag in this file: https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/flags.json


Then you can use the `useFeatures` hook to get access to the newly added flag:

https://github.com/ethyca/fides/blob/12487ea167224faa166dbfc27b1927785ee4ff90/clients/admin-ui/src/features/common/features/features.slice.ts#L141-L170